package.json

```diff
@@ -25,35 +25,36 @@
   },
   "homepage": "https://github.com/seek-oss/eslint-config-seek#readme",
   "dependencies": {
-    "@typescript-eslint/parser": "^2.10.0",
-    "babel-eslint": "^10.0.2",
-    "eslint-config-prettier": "^6.0.0",
-    "eslint-import-resolver-node": "^0.3.2",
-    "eslint-plugin-css-modules": "^2.11.0",
-    "eslint-plugin-cypress": "^2.6.0",
-    "eslint-plugin-flowtype": "^3.11.1",
-    "eslint-plugin-import": "^2.18.0",
-    "eslint-plugin-jest": "^22.10.0",
-    "eslint-plugin-react": "^7.14.2",
-    "eslint-plugin-react-hooks": "^1.6.1",
-    "find-root": "^1.1.0",
-    "typescript": "^3.5.3"
+    "@typescript-eslint/eslint-plugin": "2.10.0",
+    "@typescript-eslint/parser": "2.10.0",
```

Comment:
`2.11.0` released (yesterday?) happy to bump this up? Any reason for pinning all these deps btw?